DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateApprovals/_unapprovedDelegateExpandable.cshtml

```diff
@@ -34,7 +34,14 @@
             @Model.JobGroup
           </dd>
         </div>
-        @foreach (var customPrompt in Model.CustomPrompts) {
+        <div class="nhsuk-summary-list__row basic-summary-list__row">
+          <dt class="nhsuk-summary-list__key">
+            Professional Registration Number
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.ProfessionalRegistrationNumber" />
+        </div>
+        @foreach (var customPrompt in Model.CustomPrompts)
+        {

```

Comment:
Minor, looks like the formatter needs to be re-run as the open brace has been shifted down when it should be like it was before.